### ISSUE TITLE
Determine what patterns to add

### ISSUE THREAD

**Issue by gvn** [mozilla/mofo-bootstrap#75]
Let's work on identifying emerging patterns in our various hubs that should be added here and to mofo-ui.

What we already have added:

https://mozilla.github.io/mofo-bootstrap/demo/
https://mozilla.github.io/mofo-ui/demo/

@taisdesouzalessa since you're acquainting yourself with our work this is a good opportunity to see if you can recognize some design patterns that haven't been added already.

Let's try to meet next week to discuss our findings. @xmatthewx does that work for you?

**Comment by xmatthewx** [comment]
Works for me. Do we know all the currently sites using it?

**Comment by gvn** [comment]
- https://science.mozilla.org/ – Mofo UI & Mofo Bootstrap
- https://learning.mozilla.org/en-US/ – Mofo UI
- https://mozilla.github.io/womenandweb/ – Mofo Bootstrap
- Schedule app – uses Mofo Bootstrap...I think?

**Comment by gvn** [comment]
There may be others. If we're just visually seeing patterns between other properties we should take note of them as well...

**Comment by xmatthewx** [comment]
> Schedule app  – uses Mofo Bootstrap...I think?

Yep. I believe [Pulse](https://mzl.la/pulse) is using it as well (though maybe still on staging).

**Comment by kristinashu** [comment]
Yes, [MozFest website](https://github.com/MozillaFoundation/2016.Mozillafestival.org), [event app](https://github.com/mozilla/mozfest-event-app), and [Pulse](https://github.com/mozilla/network-pulse) all use Mofo Bootstrap but are still on staging. They aren't using much from it, but they are using it.

cc @mmmavis

**Comment by mmmavis** [comment]
Sorry @kristinashu I should've checked earlier - MozFest site isn't using mofo-bootstrap currently, it's just using Fira. However, mofo-bootstrap will be included in my [Team Page PR](https://github.com/MozillaFoundation/2016.Mozillafestival.org/issues/96) and we should be able to utilize the shared package more and remove tons of the existing MozFest site CSS. ✨

**Comment by taisdesouzalessa** [comment]
Hey @kristinashu @sabrinang @flukeout, I talked to Gavin today about the elements I should focus on first - I wanted to do a mini version of the Library first to show you and see if you think it would be useful for the team - and some questions emerged. 

I was wondering which color palette I should move to the Craft Library: the one we have on mofo-bootstrap/demo/ our the one in the Sketch file?. Although the base colors are the same, the lighter/darker versions are a bit different. Here is the side by side comparison:
![colours](https://cloud.githubusercontent.com/assets/22157921/18853086/49899a56-83f8-11e6-925f-422bc53a4a0a.jpg)

Also I was wondering if you have any examples of forms in existing pages so I can screenshot the diversity of forms and capture the different behaviors we current have?
For now, I found this one: 
<img width="756" alt="screen shot 2016-09-26 at 2 51 42 pm" src="https://cloud.githubusercontent.com/assets/22157921/18853172/b396d8b4-83f8-11e6-895e-c0116ac11c38.png">

Thanks! 

cc: @xmatthewx

**Comment by kristinashu** [comment]
Awesome! For colors, we should be using the sketch version (similar but with more greys!).

I don't think we have many forms built?

**Comment by xmatthewx** [comment]
For the colors, it also looks like the tints and shades are slightly different. Run with sketch and we can adjust bootstrap to follow.

Forms are tucked away in a few places. Definitely room for improvement and more consistency.

https://learning.mozilla.org/en-US/ > Get Updates

<img width="541" alt="screenshot 2016-09-27 14 29 49" src="https://cloud.githubusercontent.com/assets/53411/18886758/488a0b00-84bf-11e6-8ad8-aafc8004583c.png">

https://science.mozilla.org/programs/studygroups/run  

<img width="976" alt="screenshot 2016-09-27 14 30 29" src="https://cloud.githubusercontent.com/assets/53411/18886749/41d8fe7e-84bf-11e6-8e51-aa345a6f6c68.png">

https://id.webmaker.org/signup

<img width="504" alt="screenshot 2016-09-27 14 33 48" src="https://cloud.githubusercontent.com/assets/53411/18886792/713fc7d8-84bf-11e6-9c04-dbcd03c5cd5c.png">

**Comment by alanmoo** [comment]
Form elements are going to be something that's super useful to have fleshed out for upcoming work on the Science site.

**Comment by kristinashu** [comment]
Things we identified as next steps:

Designers:
- design forms https://github.com/mozilla/mofo-bootstrap/issues/93
- make an icon font https://github.com/mozilla/mofo-bootstrap/issues/91
- check mofo-ui grids https://github.com/mozilla/mofo-bootstrap/issues/90
- check mofo-ui fonts https://github.com/mozilla/mofo-bootstrap/issues/88
- add Sketch file to repo https://github.com/mozilla/mofo-bootstrap/issues/94

Engineers:
- add footer component https://github.com/mozilla/mofo-bootstrap/issues/92
- update colours https://github.com/mozilla/mofo-bootstrap/issues/89
